.github/workflows/pytest.yml

```diff
@@ -0,0 +1,37 @@
+##
+## Copyright 2021 Ocean Protocol Foundation
+## SPDX-License-Identifier: Apache-2.0
+##
+name: Operator service tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Operator Service 
+        uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        working-directory: ${{ github.workspace }}
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
+      - name: start minikube
+        id: minikube
```

Comment:
do we need minikube?

Reply:
yes, we check the connection at server startup, so all tests would fail without it. I've tried not using minikube in a previous experimental run and it failed.